golang/src/com/google/idea/blaze/golang/util/FileUtil.java

```diff
@@ -0,0 +1,19 @@
+package com.google.idea.blaze.golang.util;
+
+import java.io.File;
+
+public class FileUtil {
+    // Workaround for https://github.com/bazelbuild/intellij/issues/2057
+    // External workspace symlinks can be changed externally by practically any
+    // bazel command. Such changes to symlinks will make IntelliJ red.
+    // This helper resolves such symlink to an actual location.
```

Comment:
Please also leave a comment here that this solution uses the same approach as the one in `JarCache`. Otherwise, I would have expected to rather see symlink resolution using file APIs instead of the string manipulation this implementation does.